### PR TITLE
Update VS Code docs from the VS Code GitHub repo to DMC

### PR DESCRIPTION
**Pull request by vchapel**
Per work item 33882088

### CODE REVIEW

**Comment by codepo8** [comment]
"Selecting Launch Instance opens the browser in Visual Studio Code and displays a success message." 
That's not really true. We should replace this with "Opens a browser window and the Edge developer tools in Visual Studio Code"

**Comment by codepo8** [comment]
"The Microsoft Edge Developer Tools extension is integrated into the debugging workflow of Visual Studio Code with the Console active."
This is not part of this section though and should be moved further down into the "Browser debugging with Microsoft Edge Developer Tools integration in Visual Studio Code" - in the workflow of simply opening a browser instance you do not get the Console

**Comment by codepo8** [comment]
"Use the URL parameter to tell Visual Studio Code which URL to open."  
should be deleted as it makes no sense without explaining launch.json first. We could alternatively also state:
"Use the URL parameter inside your launch.json to tell Visual Studio Code which URL to open."

**Comment by codepo8** [comment]
"On the Activity Bar, choose Microsoft Edge Tools to open MICROSOFT EDGE TOOLS: TARGETS."

Two issues there, if we uppercase things, screenreaders will spell them, so we shouldn't do that. Also, as a sighted person I don't understand this. alternative:

"On the Activity Bar, choose Microsoft Edge Tools to open the "Microsoft Edge Tools: Targets" panel"

**Comment by codepo8** [comment]
"Choose Toggle screencast to view the browser inside the editor, or open outside in a new window if already displayed inside the editor."

that's false. it won't open in a new window, you need to do that from the target list. This button only toggles the display inside VS Code.

**Comment by codepo8** [comment]
"You can also choose Settings > Headless to open Microsoft Edge inside Visual Studio Code rather than opening the browser in a new window."

This is also not quite correct: 

"You can also choose Settings > Headless if you don't want to have an own browser window, but instead only rely on the screencast inside Visual Studio Code"

**Comment by codepo8** [comment]
"JavaScript debugging is now built in to Visual Studio Code. Starting with Version 1.5.7 of Visual Studio Code, you can start Microsoft Edge Developer Tools with the JavaScript debugger. You can debug in Chrome, Microsoft Edge, or Node.js without installing any other extensions."

One problem there is that the DevTools are only available in the Microsoft Edge case, so we need to change the order:

JavaScript debugging is now built in to Visual Studio Code. Starting with Version 1.5.7 of Visual Studio Code, you can debug in Chrome, Microsoft Edge, or Node.js without installing any other extensions. If you debug using Microsoft Edge, you can start Microsoft Edge Developer Tools with the JavaScript debugger.

**Comment by vchapel** [comment]
> "The Microsoft Edge Developer Tools extension is integrated into the debugging workflow of Visual Studio Code with the Console active."
> This is not part of this section though and should be deleted here. In the workflow of simply opening a browser instance you do not get the Console

There is an image after this text ... should it also be deleted or get new intro text?

**Comment by codepo8** [comment]
It makes no sense for it to be in that spot, let's could move it to the end
of the doc where we talk about the console.

**Comment by codepo8** [comment]
> > "The Microsoft Edge Developer Tools extension is integrated into the debugging workflow of Visual Studio Code with the Console active."
> > This is not part of this section though and should be deleted here. In the workflow of simply opening a browser instance you do not get the Console
> 
> There is an image after this text ... should it also be deleted or get new intro text?

It is repeated at the bottom of the document, where it makes more sense. So we can delete it here.

**Comment by vchapel** [comment]
Done, thanks!


[A close up of graphics  Description automatically generated]
Victor Chapel (Experis)
Technical Writer
Microsoft Edge

Alias
💻 v-vchapel
Working hours
***@***.***D757A8.ED68CE70]  11AM - 7PM (Central US)
🌉 9AM - 5PM (Pacific US)
🌍 5PM - 1AM (UTC+1)
Pronouns
🆗 he/him  🆗 they/them

File: microsoft-edge/visual-studio-code/microsoft-edge-devtools-extension.md [hunk 1]
```diff
@@ -1,133 +1,211 @@
...
 ms.prod: microsoft-edge
-keywords: microsoft edge, web development, f12 tools, devtools, vs code, visual studio code, elements
+keywords: microsoft edge, web development, f12 tools, devtools, vs code, visual studio code, microsoft edge developer tools, microsoft edge developer tools extension
 ---
 
-# Microsoft Edge DevTools for Visual Studio Code extension  
+# Microsoft Edge Developer Tools for Visual Studio Code extension  
 
-Use <!--the [Microsoft Edge DevTools for Visual Studio Code][VisualstudioMarketplaceElementsMicrosoftEdgeChromium] -->this extension to access in Microsoft Edge DevTools inside [Microsoft Visual Studio Code][VisualstudioCode].  You have access to the **Elements** and **Network** tools in Microsoft Edge DevTools.  You may either launch or attach to an instance of Microsoft Edge.  After you connect you may display the runtime HTML structure, alter the layout, fix styling issues, and inspect network traffic.  
+The Microsoft Edge Developer Tools for Visual Studio Code extension enables you to use the browser’s Elements and Network tool from within the editor.
```
Comment (Reezaali) [inline microsoft-edge/visual-studio-code/microsoft-edge-devtools-extension.md:14]:
```suggestion
The Microsoft Edge Developer Tools for Visual Studio Code extension allows you to use the browser’s Elements and Network tool from within the editor. 
```

File: microsoft-edge/visual-studio-code/microsoft-edge-devtools-extension.md [hunk 2]
```diff
@@ -1,133 +1,211 @@
...
+keywords: microsoft edge, web development, f12 tools, devtools, vs code, visual studio code, microsoft edge developer tools, microsoft edge developer tools extension
 ---
 
-# Microsoft Edge DevTools for Visual Studio Code extension  
+# Microsoft Edge Developer Tools for Visual Studio Code extension  
 
-Use <!--the [Microsoft Edge DevTools for Visual Studio Code][VisualstudioMarketplaceElementsMicrosoftEdgeChromium] -->this extension to access in Microsoft Edge DevTools inside [Microsoft Visual Studio Code][VisualstudioCode].  You have access to the **Elements** and **Network** tools in Microsoft Edge DevTools.  You may either launch or attach to an instance of Microsoft Edge.  After you connect you may display the runtime HTML structure, alter the layout, fix styling issues, and inspect network traffic.  
+The Microsoft Edge Developer Tools for Visual Studio Code extension enables you to use the browser’s Elements and Network tool from within the editor. 
+Without leaving Visual Studio Code, use Microsoft Edge Developer Tools (DevTools) to connect to an instance of Microsoft Edge with the following abilities.
```
Comment (Reezaali) [inline microsoft-edge/visual-studio-code/microsoft-edge-devtools-extension.md:15]:
```suggestion
Without leaving Visual Studio Code, use Microsoft Edge Developer Tools (DevTools) to connect to an instance of Microsoft Edge with the following functionality.
```

File: microsoft-edge/visual-studio-code/microsoft-edge-devtools-extension.md [hunk 3]
```diff
@@ -1,133 +1,211 @@
...
+* See the runtime HTML structure.
```
Comment (Reezaali) [inline microsoft-edge/visual-studio-code/microsoft-edge-devtools-extension.md:16]:
```suggestion
* View the runtime HTML structure.
```

File: microsoft-edge/visual-studio-code/microsoft-edge-devtools-extension.md [hunk 4]
```diff
@@ -1,133 +1,211 @@
...
 
-By default, the extension launches a browser instance in a unique window and gives you the **Elements** functionality of Microsoft Edge DevTools.  
+There are two ways to open Microsoft Edge Developer Tools.
+* From the activity bar in Visual Studio Code.
+* From the JavaScript Debugger in Visual Studio Code.
 
-:::image type="complex" source="./media/edge-devtools-for-vscode-windowed.png" alt-text="Microsoft Edge DevTools for Visual Studio Code running with a full browser window" lightbox="./media/edge-devtools-for-vscode-windowed.png":::
-   Microsoft Edge DevTools for Visual Studio Code running with a full browser window  
-:::image-end:::  
+This article describes the Microsoft Edge Developer Tools for Visual Studio Code extension. You can [download the extension][https://marketplace.visualstudio.com/items?itemName=ms-edgedevtools.vscode-edge-devtools] and [check the source code][https://github.com/microsoft/vscode-edge-devtools].
```
Comment (Reezaali) [inline microsoft-edge/visual-studio-code/microsoft-edge-devtools-extension.md:29]: Can we fix the links? [pagename][linkname] and then add the link details in the section at the bottom of the page.

File: microsoft-edge/visual-studio-code/microsoft-edge-devtools-extension.md [hunk 5]
```diff
@@ -1,133 +1,211 @@
...
-In the extension settings, you may enable more functionality like **Headless Mode** and **Network**.  
+## Opening Microsoft Edge Developer Tools with Visual Studio Code  
 
-:::image type="complex" source="./media/edge-devtools-for-vscode-settings.png" alt-text="Enabling (or disabling) headless mode and Network inspection in the extension settings" lightbox="./media/edge-devtools-for-vscode-settings.png":::
-   Enabling \(or disabling\) headless mode and Network inspection in the extension settings  
-:::image-end:::  
+To open the tools panel, choose the Microsoft Edge Tools icon from the Activity bar.
 
-## Headless Mode  
+The Microsoft Edge Developer Tools for Visual Studio Code extension enables you to launch an instance of Edge or generate a `launch.json` file to automate the debugging workflow in the future.
```
Comment (Reezaali) [inline microsoft-edge/visual-studio-code/microsoft-edge-devtools-extension.md:35]:
```suggestion
The Microsoft Edge Developer Tools for Visual Studio Code extension allows you to launch an instance of Edge or generate a `launch.json` file to automate the debugging workflow. 
```

File: microsoft-edge/visual-studio-code/microsoft-edge-devtools-extension.md [hunk 6]
```diff
@@ -1,133 +1,211 @@
...
+Selecting **Launch Instance** opens a browser window and the Edge developer tools in Visual Studio Code.   
 
-> [!NOTE]
-> On macOS, you should turn on headless mode as your preferred mode.  Using headless mode should solve an issue where, if the window is not visible, the browser window reports that it is `Not Active`.  
+:::image type="complex" source="./media/edge-devtools-for-vscode-launch-instance.png" alt-text="Select Launch Instance to open the browser in Visual Studio Code" lightbox="./media/edge-devtools-for-vscode-launch-instance.png":::
+   Select Launch Instance to open the browser in Visual Studio Code  
+:::image-end:::
 
-## Network tool  
+Use the Developer Tools extension in VS Code to inspect an HTML Element in Microsoft Edge. For example, select **Success!** In the browser and notice that the Elements tool opens with the HTML highlighted.
```
Comment (Reezaali) [inline microsoft-edge/visual-studio-code/microsoft-edge-devtools-extension.md:47]:
```suggestion
Use the Developer Tools extension in VS Code to inspect an HTML Element in Microsoft Edge. For example, select **Success!** in the browser and notice that the Elements tool opens with the HTML highlighted. 
```

File: microsoft-edge/visual-studio-code/microsoft-edge-devtools-extension.md [hunk 7]
```diff
@@ -1,133 +1,211 @@
...
+:::image-end:::
 
-:::image type="complex" source="./media/edge-devtools-for-vscode-network.png" alt-text="Network inspection in Microsoft Edge DevTools for Visual Studio Code" lightbox="./media/edge-devtools-for-vscode-network.png":::
-    Network inspection in Microsoft Edge DevTools for Visual Studio Code  
-:::image-end:::  
+## Using Microsoft Edge Developer Tools in Visual Studio Code  
 
-## Launching Microsoft Edge From the extension  
+You can use this extension in one of three modes:
+* Launch Microsoft Edge in a new window navigated to your app.
```
Comment (Reezaali) [inline microsoft-edge/visual-studio-code/microsoft-edge-devtools-extension.md:56]:
```suggestion
* Launch Microsoft Edge in a new window, and navigate to your web application.
```

File: microsoft-edge/visual-studio-code/microsoft-edge-devtools-extension.md [hunk 8]
```diff
@@ -1,133 +1,211 @@
...
+* Attach to a running instance of Microsoft Edge.
+* Open a new instance of Microsoft Edge inside Visual Studio Code. 
 
-Navigate to Microsoft Edge Tools in the **Activity Bar**.  Next to where it says **Microsoft Edge Tools: Targets,** there is a plus sign that opens the browser for your app.  If you choose the **about:blank** option, you must navigate to your web app in the browser for it to appear in the **Elements** panel in Visual Studio Code.  
+Each mode requires you to serve your web application from a local web server, which is started from either a Visual Studio Code task or from the command line. Use the URL parameter inside the `launch.json` file to tell Visual Studio Code which URL to open.
 
-## Launching Microsoft Edge from the Debug view  
+### Open a new browser instance
```
Comment (Reezaali) [inline microsoft-edge/visual-studio-code/microsoft-edge-devtools-extension.md:62]: Can we add an intro sentence below this heading and before the numbered steps?

File: microsoft-edge/visual-studio-code/microsoft-edge-devtools-extension.md [hunk 9]
```diff
@@ -1,133 +1,211 @@
...
+
+By default, the extension launches Microsoft Edge in a new window, which displays another browser icon on the task bar.
+
+Choose **Toggle screencast** to display the browser inside the editor, or hide the browser if already displayed. 
+
+:::image type="complex" source="./media/edge-devtools-for-vscode-toggle-screencast.png" alt-text="Toggle screencast to view the browser inside the editor" lightbox="./media/edge-devtools-for-vscode-toggle-screencast.png":::
+   Toggle screencast to view the browser inside the editor 
+:::image-end:::
+
+You can also choose **Settings** > **Headless** to use only the screencast browser inside Visual Studio Code.
```
Comment (Reezaali) [inline microsoft-edge/visual-studio-code/microsoft-edge-devtools-extension.md:116]: Can we include the full UI text for the headless option?

File: microsoft-edge/visual-studio-code/microsoft-edge-devtools-extension.md [hunk 10]
```diff
@@ -1,133 +1,211 @@
...
+## Opening source files from the Elements tool
+One of the features of the Elements tool is that it displays the source file that applied styles and event handlers to a selected node in the DOM tree. Source files appear in the form of links to a URL. Selecting a link opens that file in the Visual Studio Code editor.
+
+:::image type="complex" source="./media/edge-devtools-for-vscode-elements-files.png" alt-text="Open source files from Elements tool" lightbox="./media/edge-devtools-for-vscode-elements-files.png":::
+   Open source files from Elements tool  
+:::image-end:::
+
+## Setting up your project to show live changes in the extension
+By default, the extension does not track live changes to the code as you write. If you want the browser to automatically refresh when you change a file, set up a live reload environment. This requires [Node.js][https://www.nodejs.org/] and [NPM][https://www.npmjs.com/package/reload?activeTab=readme] on your machine.
+The following example shows a folder of production files on your hard drive called `my-project`. Change the file names and locations as needed for your environment.
```
Comment (Reezaali) [inline microsoft-edge/visual-studio-code/microsoft-edge-devtools-extension.md:131]: Can we fix the links?

File: microsoft-edge/visual-studio-code/microsoft-edge-devtools-extension.md [hunk 11]
```diff
@@ -1,133 +1,211 @@
...
+
+1. Install Node.js and the reload NPM package.
+    1. Download and install [Node.js][https://www.nodejs.org/].
+    1. To install the [reload NPM package][https://www.npmjs.com/package/reload?activeTab=readme], open a command prompt and run `npm install reload -g` to install the package globally.
```
Comment (Reezaali) [inline microsoft-edge/visual-studio-code/microsoft-edge-devtools-extension.md:135]: Can we fix the links?

File: microsoft-edge/visual-studio-code/microsoft-edge-devtools-extension.md [hunk 12]
```diff
@@ -1,133 +1,211 @@
...
+1. Attach the extension to your live reloading project.
+    1. Navigate to the `my-project` folder in your terminal window and run `reload` to start the local server.
+    1. In Visual Studio Code, open the `my-project` folder.
+    1. Go to the extension and launch a Microsoft Edge browser instance.
+    1. In Microsoft Edge, navigate the extension to `localhost:8080/{file name you want to open}`.
+    1. All changes saved in this folder now trigger a browser refresh.
+
+## Browser debugging with Microsoft Edge Developer Tools integration in Visual Studio Code
+JavaScript debugging is now built in to Visual Studio Code. Starting with Version 1.5.7 of Visual Studio Code, you can debug in Chrome, Microsoft Edge, or Node.js without installing any other extensions. If you debug using Microsoft Edge, you can start Microsoft Edge Developer Tools with the JavaScript debugger.
+1. To start a session:
```
Comment (Reezaali) [inline microsoft-edge/visual-studio-code/microsoft-edge-devtools-extension.md:145]:
```suggestion
1. To start a session use either of the following methods.
```

File: microsoft-edge/visual-studio-code/microsoft-edge-devtools-extension.md [hunk 13]
```diff
@@ -1,133 +1,211 @@
...
+
+1. Choose **Edge**.
+    On the debug toolbar, notice the new Inspect button 
+
+    :::image type="complex" source="./media/edge-devtools-for-vscode-inspect-button.png" alt-text="Inspect button now displayed on debug toolbar" lightbox="./media/edge-devtools-for-vscode-inspect-button.png":::
+       Inspect button now displayed on debug toolbar 
+    :::image-end:::
+
+1. Choose **Inspect** to open Microsoft Edge Developer Tools inside Visual Studio Code.
+    The first time you choose **Inspect**, the editor prompts you to install the [Microsoft Edge Developer Tools for Visual Studio Code extension][https://marketplace.visualstudio.com/items?itemName=ms-edgedevtools.vscode-edge-devtools].
```
Comment (Reezaali) [inline microsoft-edge/visual-studio-code/microsoft-edge-devtools-extension.md:161]: can we fix the link?

File: microsoft-edge/visual-studio-code/microsoft-edge-devtools-extension.md [hunk 14]
```diff
@@ -1,133 +1,211 @@
...
+    After installation is complete, when you choose **Inspect** it opens Microsoft Edge Developer Tools inside Visual Studio Code.
+
+    :::image type="complex" source="./media/edge-devtools-for-vscode-tools-inside.png" alt-text="Inspect button opens Microsoft Edge Developer Tools inside Visual Studio Code" lightbox="./media/edge-devtools-for-vscode-tools-inside.png":::
+       Inspect button opens Microsoft Edge Developer Tools inside Visual Studio Code
+    :::image-end:::
+
+    Now you can inspect the DOM, change CSS, and see network requests of your project running in the browser without leaving Visual Studio Code.
     
-    ```json
+    You can also use the Debug Console in the editor to interact with the document in the browser. You have full access to the window object and can use the [Console Utilities API][https://docs.microsoft.com/microsoft-edge/devtools-guide-chromium/console/utilities].
```
Comment (Reezaali) [inline microsoft-edge/visual-studio-code/microsoft-edge-devtools-extension.md:171]: can we fix the link?

File: microsoft-edge/visual-studio-code/microsoft-edge-devtools-extension.md [hunk 15]
```diff
@@ -1,133 +1,211 @@
...
+
+    :::image type="complex" source="./media/edge-devtools-for-vscode-debug-console.png" alt-text="Debug Console in editor interacts with document open in browser" lightbox="./media/edge-devtools-for-vscode-debug-console.png":::
+       Debug Console in editor interacts with document open in browser  
+    :::image-end:::
+
+1. To automatically attach to Microsoft Edge and launch Developer Tools in Visual Studio Code, create a [launch.json][https://code.visualstudio.com/Docs/editor/debugging#_launch-configurations] file.
```
Comment (Reezaali) [inline microsoft-edge/visual-studio-code/microsoft-edge-devtools-extension.md:177]: can we fix the link?

Comment (Reezaali) [reply microsoft-edge/visual-studio-code/microsoft-edge-devtools-extension.md:177]: can we change to...

create a launch.json file. For more information, navigate to...

File: microsoft-edge/visual-studio-code/microsoft-edge-devtools-extension.md [hunk 16]
```diff
@@ -1,133 +1,211 @@
...
-    *   Select `F5`.  
-    *   Choose the **Play** \(green\) button.  
-         
-Now, you may do the following actions.  
-
-*   Access a screencast of your browser.  
-*   Inspect the DOM and the styling of the components on your page.  
     
+## Console integration
+If you launch the extension from the Run and Debug workflow, the [Debug Console of Visual Studio Code][https://code.visualstudio.com/Docs/editor/debugging] gives you all the functions of the Microsoft Edge Developer Tools [Console][https://docs.microsoft.com/microsoft-edge/devtools-guide-chromium/console/]. You have direct access to the Window object of the instance of Edge and can use all the [utility methods][https://docs.microsoft.com/microsoft-edge/devtools-guide-chromium/console/utilities].
```
Comment (Reezaali) [inline microsoft-edge/visual-studio-code/microsoft-edge-devtools-extension.md:198]: Can we fix the links?

File: microsoft-edge/visual-studio-code/microsoft-edge-devtools-extension.md [hunk 17]
```diff
@@ -1,133 +1,211 @@
...
+
+:::image type="complex" source="./media/edge-devtools-for-vscode-console-integration.png" alt-text="Microsoft Edge Developer Tools Console available when extension launched from Run and Debug workflow" lightbox="./media/edge-devtools-for-vscode-console-integration.png":::
+   Microsoft Edge Developer Tools Console available when extension launched from Run and Debug workflow  
+:::image-end:::
+
 ## Getting in touch with the Microsoft Edge DevTools for Visual Studio Code extension team  
 
 Send your feedback by [filing an issue][GithubMicrosoftVscodeEdgeDevtoolsNewIssue] against the [GitHub repo][GithubMicrosoftVscodeEdgeDevtools] of the extension.
```
Comment (Reezaali) [inline microsoft-edge/visual-studio-code/microsoft-edge-devtools-extension.md:206]:
```suggestion
Send your feedback by [filing an issue][GithubMicrosoftVscodeEdgeDevtoolsNewIssue] on the [microsoft/vscode-edge-devtools GitHub repo][GithubMicrosoftVscodeEdgeDevtools] of the extension.  
```

File: microsoft-edge/visual-studio-code/microsoft-edge-devtools-extension.md [hunk 18]
```diff
@@ -1,133 +1,211 @@
...
+:::image type="complex" source="./media/edge-devtools-for-vscode-console-integration.png" alt-text="Microsoft Edge Developer Tools Console available when extension launched from Run and Debug workflow" lightbox="./media/edge-devtools-for-vscode-console-integration.png":::
+   Microsoft Edge Developer Tools Console available when extension launched from Run and Debug workflow  
+:::image-end:::
+
 ## Getting in touch with the Microsoft Edge DevTools for Visual Studio Code extension team  
 
 Send your feedback by [filing an issue][GithubMicrosoftVscodeEdgeDevtoolsNewIssue] against the [GitHub repo][GithubMicrosoftVscodeEdgeDevtools] of the extension.  
 
-If you want to help make <!--the Microsoft Edge DevTools for Visual Studio Code -->this extension better, your contributions are welcome.  Find everything you need to get started in the [GitHub repo][GithubMicrosoftVscodeEdgeDevtools] of the extension.  
+If you want to help make <!--the Microsoft Edge DevTools for Visual Studio Code -->this extension better, your contributions are welcome.  Find everything you need to get started in the [GitHub repo][GithubMicrosoftVscodeEdgeDevtools] of the extension.
```
Comment (Reezaali) [inline microsoft-edge/visual-studio-code/microsoft-edge-devtools-extension.md:208]:
```suggestion
If you want to help make <!--the Microsoft Edge DevTools for Visual Studio Code -->this extension better, your contributions are welcome.  Find everything you need to get started in the [microsoft/vscode-edge-devtools GitHub repo][GithubMicrosoftVscodeEdgeDevtools] of the extension. 
```

**Review by Reezaali — COMMENTED** [review]
Some suggestions and edits. Please review.

**Comment by codepo8** [comment]
LGTM, let's ship it. I will then redirect the docs on Monday.

**Review by codepo8 — APPROVED** [review]

**Pull request merged**